File: javascript-codes/fizzBuzz.js

const prompt = require('prompt-sync')();

function fizzBuzz(number){
    let ctr = 1;
    while (ctr <= number){
        if (ctr%3 == 0){
            let string = "Fizz"
            if (ctr%5 == 0){
                string = string + "Buzz"
            } 
            console.log(string)
        } else if (ctr%5 == 0){
            console.log("Buzz")
        } else {
            console.log(ctr)
        }
        ctr++;
    }
}



let input = (prompt("Enter the number: "));

let number = parseInt(input);
if (isNaN(number)){
    console.log("Invalid input");
    process.exit(1)
}
console.log(number);
if (number < 0 || number > 100){
    console.log("Invalid number. Please enter number from 1-100.");
    process.exit(1)
}

fizzBuzz(number);



    

File: javascript-codes/fizzbuzz.py
# This source file is meant for the fizzBuzz


def fizz_buzz(number):
    ctr=1
    while ctr <= number:
        if ctr%3==0:
            print("Fizz", end="")
            if ctr%5==0:
                print("Buzz", end="")
        elif ctr%5==0:
            print("Buzz", end="")
        else:
            print(ctr, end="")
        print()
        ctr+=1
        
def ask_for_input():
    try:
        number = int(input("Enter your number:"))
        print(number)
        if number < 0 and number > 100:
            print("Please enter a number more within 1-100")
            ask_for_input()
        fizz_buzz(number)
    except ValueError as err:
        print('Please enter a valid input number')
        ask_for_input()
    
        
if __name__ == "__main__":
    ask_for_input()
    
        